draft-ietf-ppm-dap.md

```diff
@@ -1480,62 +1539,39 @@ An `AggregationJobResp` is valid only if it satisfies the following requirement:
 
 * The Helper's `prepare_steps` MUST include exactly the same report IDs in the
   same order as either the `report_shares` in the Leader's
-  `AggregationJobInitReq` (if this is the first round of continuation) or the
+  `AggregationJobInitReq` (if this is the first round of aggregation) or the
   `prepare_steps` in the Leader's `AggregationJobContinueReq` (if this is a
   subsequent round).
 
 [[OPEN ISSUE: consider relaxing this ordering constraint. See issue#217.]]
 
 ### Aggregate Continuation {#agg-continue-flow}
 
-In the continuation phase, the leader drives the VDAF preparation of each share
+In the continuation phase, the Leader drives the VDAF preparation of each share
 in the candidate report set until the underlying VDAF moves into a terminal
-state, yielding an output share for all Aggregators or an error. This phase may
+state, yielding an output share for all aggregators or an error. This phase may
 involve multiple rounds of interaction depending on the underlying VDAF. Each
-round trip is initiated by the leader.
+round trip is initiated by the Leader.
 
 #### Leader Continuation {#aggregation-leader-continuation}
 
-The Leader begins each round of continuation for a report share based on its
-locally computed prepare message and the previous PrepareStep from the Helper.
-If PrepareStep is of type `failed`, then the leader acts based on the value of
-the `ReportShareError`:
-
-  - If the error is `ReportShareError.report_too_early`, then the leader MAY try
-    to re-send the report in a later `AggregationJobInitReq`.
-  - For any other error, the leader marks the report as failed, removes it from
-    the candidate report set and does not process it further.
```

Comment:
I don't think the new text discusses what the leader should do if the `PrepareStep` is of type `failed`.